src/test/java/com/wikia/webdriver/common/templates/fandom/AdsFandomTestTemplate.java

```diff
@@ -0,0 +1,15 @@
+package com.wikia.webdriver.common.templates.fandom;
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.url.FandomUrlBuilder;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.url.FandomUrlBuilder'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)